omim2obo/main.py

```diff
@@ -164,21 +198,28 @@ def omim2obo(use_cache: bool = False):
                 continue
 
         # - Non-deprecated
-        # Parse titles
-        omim_type, pref_labels_str, alt_labels, inc_labels = omim_type_and_titles[omim_id]
-        other_labels = []
-        cleaned_inc_labels = []
-        label_endswith_included_alt = False
-        label_endswith_included_inc = False
-        pref_labels: List[str] = [x.strip() for x in pref_labels_str.split(';')]
-        pref_title: str = pref_labels[0]
-        pref_symbols: List[str] = pref_labels[1:]
-        if alt_labels:
-            cleaned_alt_labels, label_endswith_included_alt = get_alt_labels(alt_labels)
-            other_labels += cleaned_alt_labels
-        if inc_labels:
-            cleaned_inc_labels, label_endswith_included_inc = get_alt_labels(inc_labels)
-            # other_labels += cleaned_inc_labels  # deactivated 7/2024 in favor of alternative for tagging 'included'
+        # Parse titles & symbols
+        omim_type, pref_titles_str, alt_titles_str, inc_titles_str = omim_type_and_titles[omim_id]
+        alt_titles: List[str] = []
+        alt_symbols: List[str] = []
+        former_alt_titles: List[str] = []
+        former_alt_symbols: List[str] = []
+        included_titles: List[str] = []
+        included_symbols: List[str] = []
+
+        pref_titles: List[str] = [x.strip() for x in pref_titles_str.split(';')]
+        pref_title: str = pref_titles[0]
+        pref_symbols: List[str] = pref_titles[1:]
+        if alt_titles_str:
+            alt_titles, alt_symbols = parse_title_symbol_pairs(alt_titles_str)
+            alt_titles, alt_symbols, former_alt_titles, former_alt_symbols = \
+                separate_former_titles_and_symbols(alt_titles, alt_symbols)
+            alt_titles, alt_symbols = clean_alt_and_included_titles(alt_titles, alt_symbols)
+            former_alt_titles, former_alt_symbols = clean_alt_and_included_titles(former_alt_titles, former_alt_symbols)
+        if inc_titles_str:
+            included_titles, included_symbols = parse_title_symbol_pairs(inc_titles_str)
+            included_titles, included_symbols = clean_alt_and_included_titles(included_titles, included_symbols)
+        included_is_included = included_titles or included_symbols  # redundant. can't be included symbol w/out title
```

Comment:
are you doing this so you don't have to check for an empty list to see if the entry has any values in the "Included Title(s); symbols" column?

Reply:
Naw. The reason I did this is to make the code clearer. Didn't want to come across the code just checking for "included titles" and have someone thing "well why isn't it checking for included symbols?".

Come to think of it, another approach is that I could remove this line. And instead, further down, I could potentially replace `if included_is_included` with something like `if included_titles:  # fyi: only need to check for included titles because there can't be any included symbols unless there are included titles`.

Reply:
yes, it is a somewhat oddly named variable and duplicates information, `included_titles`, that already exists in this case

Reply:
Done!